test/e2e/pre_provisioning_test.go

```diff
@@ -214,6 +214,53 @@ var _ = ginkgo.Describe("Pre-Provisioned", func() {
 			}
 			test.Run(cs, ns)
 		})
+
+		ginkgo.It("should succeed when reattaching a disk to a new node on DanglingAttachError [disk.csi.azure.com][shared disk]", func() {
+			skipIfUsingInTreeVolumePlugin()
+			skipIfOnAzureStackCloud()
+			sharedDiskSize := int64(1024)
+			req := makeCreateVolumeReq("reattach-disk-multiple-pods", sharedDiskSize)
+			req.Parameters = map[string]string{
+				"skuName":     "Premium_LRS",
+				"maxShares":   "5",
```

Comment:
remove this `"maxShares":   "5",`,  this test should be only for non-shared disk